src/pint/utils.py

```diff
@@ -1686,6 +1694,144 @@ def remove_dummy_distance(c):
         return c
 
 
+def info_string(prefix_string="# ", comment=None):
+    """Returns an informative string about the current state of PINT.
+
+    Adds:
+
+    - Creation date
```

Comment:
If you delete the spaces between lines this should render as an actual list in Sphinx.

Reply:
I think it's correct as-is.  The version at https://nanograv-pint--1069.org.readthedocs.build/en/1069/api/pint.utils.html#pint.utils.info_string seems outdated (maybe because CI is failing because of coverage).  But the version I build locally looks OK.

<img width="764" alt="Screen Shot 2021-07-22 at 10 34 32 AM" src="https://user-images.githubusercontent.com/5092062/126666885-29d34afa-3a06-4a8b-8cdd-40fa82573066.png">

Reply:
Coverage doesn't prevent other jobs, and readthedocs seems to be mishandling this. Could there be a Sphinx version difference? Unfortunately the version that people actually read online is the readthedocs version.

Reply:
If there is a version issue, we should perhaps pin that in the development requirements.  I had found versions of the documentation that required blank lines between list elements, but I also found the opposite. So we'll try it the other way...

Reply:
I don't know what's going on. I'm going to assume it's fine now and readthedocs has a caching problem.